### PR TITLE
chore(parse-i3s-tile-content): add types

### PR DESCRIPTION
**Pull request by HeimEndyd**

### CODE REVIEW

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 1]
```diff
@@ -82,25 +90,20 @@ export async function parseI3STileContent(
...
   if (!tile.content) {
     return tile;
   }
 
   const content = tile.content;
-  let attributes;
-  let vertexCount;
-  let byteOffset = 0;
-  let featureCount = 0;
+  let attributes: normalizedAttributes;
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:103]:
```suggestion
  let attributes: NormalizedAttributes;
```

File: modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts [hunk 2]
```diff
@@ -151,7 +154,7 @@ async function parseI3SNodeGeometry(
       byteOffset,
       vertexAttributes,
       vertexCount,
-      attributesOrder
+      attributesOrder!
```
Comment (ibgreen) [inline modules/i3s/src/lib/parsers/parse-i3s-tile-content.ts:157]:
```suggestion
      // @ts-expect-error
      attributesOrder
```

File: modules/i3s/src/types.ts [hunk 3]
```diff
@@ -311,24 +336,78 @@ type Store = {
   lodModel: string;
   defaultGeometrySchema: DefaultGeometrySchema;
 };
-
+// Spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.8/defaultGeometrySchema.cmn.md
```
Comment (ibgreen) [inline modules/i3s/src/types.ts:339]: Use JSDoc notation 
```suggestion
/** Spec - https://github.com/Esri/i3s-spec/blob/master/docs/1.8/defaultGeometrySchema.cmn.md */
```

**Review by ibgreen — APPROVED** [review]

**Comment by HeimEndyd** [comment]
These changes were implemented within https://github.com/visgl/loaders.gl/pull/1911

**Pull request closed without merging**